local-tests/chainrunner/node.py

```diff
@@ -65,6 +71,13 @@ def highest_block(self):
             return int(a), int(b)
         return -1, -1
 
+    def check_hash_of(self, number):
```

Comment:
nit: I'd suggest a bit different naming here. This doesn't check anything, it's a getter. How about `get_hash_of` or (even better) `get_hash`? Also the argument would be more self-explanatory if it was called `height`

Reply:
done, thx